models/edu/stanford/nlp/models/sutime/english.sutime.txt

```diff
@@ -888,37 +898,52 @@
     result: $0[-1].temporal.value
   }
-
   { name: "temporal-composite-7a",
     pattern: ( /every/ ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
-	result: MakePeriodicTemporalSet($1[0].temporal, "every", 1 )
+	  result: MakePeriodicTemporalSet($1[0].temporal, "every", 1 )
   }
-
   { name: "temporal-composite-7b",
-#    pattern: ( ( $FREQ_MOD ) ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
+#   pattern: ( ( $FREQ_MOD ) ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
     pattern: ( ( $FREQ_MOD ) ( [ $hasTemporal ] ) ),
-	result: MakePeriodicTemporalSet($2[0].temporal, GetTag($1[0], "PTS.quant"), GetTag($1[0], "PTS.multiple") )
+	  result: MakePeriodicTemporalSet($2[0].temporal, GetTag($1[0], "PTS.quant"), GetTag($1[0], "PTS.multiple") )
+  }
+  { name: "temporal-composite-8a",
+    active: options."markTimeRanges",
+    pattern: ( /from|between/ /the/? ($INTORD) /to|and|through|-/ ( [ { temporal::IS_TIMEX_TIME } | { temporal::IS_TIMEX_DATE } ] ) ),
+	  result: TimeRange( $1[0].numcompvalue, $2[0].temporal.value.month, $2[0].temporal.value )
+  }
+  { name: "temporal-composite-8b",
+	  active: options."markTimeRanges",
+    pattern: ( /from|between/ ( [ { temporal::IS_TIMEX_TIME } | { temporal::IS_TIMEX_DATE } ] ) /to|and|through|-/ ( [ { temporal::IS_TIMEX_TIME } |   {   temporal::IS_TIMEX_DATE } ] ) ),
+	  result: TimeRange( $1[0].temporal.value, $2[0].temporal.value )
   }
-
-  { name: "temporal-composite-8:ranges",
-	active: options."markTimeRanges",
-    pattern: ( /from|between/? ( [ { temporal::IS_TIMEX_TIME } | { temporal::IS_TIMEX_DATE } ] ) /to|and|through|-/ ( [ { temporal::IS_TIMEX_TIME } |   {   temporal::IS_TIMEX_DATE } ] ) ),
-	result: TimeRange( $1[0].temporal.value, $2[0].temporal.value ) }
-
   { name: "temporal-composite-9",
-	pattern: ( [{ temporal::IS_TIMEX_TIME }] (?: /sharp/|/exactly/|/precisely/|/on/ /the/ /dot/) ),
-    result: $0[0].temporal.value }
+	  pattern: ( [{ temporal::IS_TIMEX_TIME }] (?: /sharp/|/exactly/|/precisely/|/on/ /the/ /dot/) ),
+    result: $0[0].temporal.value
+  }
+  { name: "temporal-composite-10",
+	  pattern: ( /since/ ( [ { temporal::IS_TIMEX_DATE } ] ) ),
+    result: TimeRange( $1[0].temporal.value, NIL, 0, 0 )
+  }
+  { name: "temporal-composite-11",
+	  pattern: ( ( /after/ | /later/ /than/ ) /the/? ( [ { temporal::IS_TIMEX_DATE } ] ) ),
+    result: TimeRange( $2[0].temporal.value, NIL, 1, 0 )
+  }
+  { name: "temporal-composite-12",
+	  pattern: ( ( /before/ | /prior/ /to/ | /until/ ) /the/? ( [ { temporal::IS_TIMEX_DATE } ] ) ),
+    result: TimeRange( NIL, $2[0].temporal.value, 0, 1 )
+  }
 
   ########################################################################################################################
 
   ENV.defaults["stage"] = 4
   ENV.defaults["ruleType"] = "tokens"
 
   # Vague times
-  { ( /the/ /past/ | /newly/ | /not/ /long/ /ago/ | /these/ /days/) => TIME_PAST }
+  { ( /the/ /past/ | /recently/ ) => TIME_PAST }
```

Comment:
Are the old keywords not needed anymore?

Reply:
These were removed by mistake when I merged in Stanford's latest version of the file where they were not contained. I've added them back.